README.md

```diff
@@ -5,7 +5,18 @@ Exercism Exercises in Elm
 
 ## Setup
 
-Detailed installation instructions can be found at [http://elm-lang.org/install](http://elm-lang.org/install).
+- Detailed installation instructions for elm can be found at [http://elm-lang.org/install](http://elm-lang.org/install).
+
+- Install `elm-test` for the command line with `npm install elm-test`.
+
+### Running Tests in an Exercise Directory
+Inside an exercise directory, Elm must be configured manually.
+
+- Install `laszlopandy/elm-console` and `deadfoxygrandpa/elm-test`.
+  - `elm-package install -y laszlopandy/elm-console`
+  - `elm-package install -y deadfoxygrandpa/elm-test`
+- You should then be able to run tests with `elm-test ExerciseModuleNameTest.elm`
+
```

Comment:
yeah, this is really more about how users will have to run the exercises once they have pulled one down. I think it maybe goes somewhere else eventually but thought it should go here as well.